framework/runtime.cc

```diff
@@ -57,7 +54,7 @@ Initialize()
   SystemWideEventPublisher::GetInstance().PublishEvent(Event("ProgramStart"));
 
   // Disable internal HDF error reporting
-  H5Eset_auto2(H5E_DEFAULT, NULL, NULL);
+  H5Eset_auto2(H5E_DEFAULT, nullptr, nullptr);
```

Comment:
We should be cautious here. We are using the `C` interface for `HDF5`, and `nullptr` is not always a drop-in replacement for `NULL`. While it's likely to work most of the time, there are specific instances where things can go awry. If we are using the `C` interface to a third-party library, we should probably stick with `C` constructs and use `NULL`.